corehq/apps/locations/tests/test_location_queries.py

```diff
@@ -181,21 +181,23 @@ class TestFilterPath(LocationHierarchyTestCase):
 
     def test_path_queries(self):
         for querystring, expected in [
-            ('Suffolk', ['Suffolk', 'Boston', 'Brookline']),
-            ('Suff', ['Suffolk', 'Boston', 'Brookline']),
+            ('Suff', ['Suffolk']),
```

Comment:
Nit: Might be nice to show what an empty `querystring` does:
```python
            ('', ['Massachusetts', 'Middlesex', 'Cambridge', 'Somerville', 'Evil Somerville',
                  'Suffolk', 'Boston', 'Brookline', 'Copycat State', 'Evil Middlesex', 'Cambridge',
                  'Somerville'])
```

Reply:
https://github.com/dimagi/commcare-hq/pull/26699